build/version.go

```diff
@@ -39,7 +39,7 @@ func BuildTypeString() string {
 }
 
 // NodeBuildVersion is the local build version of the Lotus daemon
-const NodeBuildVersion string = "1.27.1"
+const NodeBuildVersion string = "1.27.2-dev"
```

Comment:
Are we sure we don't need to change `MinerBuildVersion ` here @rjan90  ? I think not because I don't see any Miner related changes in this PR but wanted to confirm.